ffv1.md

```diff
@@ -429,24 +429,60 @@ Early experimental versions of FFV1 used the CABAC Arithmetic coder from H.264 a
 
 #### Range Binary Values
 
-To encode binary digits efficiently a Range coder is used.
+To encode binary digits efficiently a Range coder is used. A Range coder encodes a series of symbols by using a probability estimation for each symbol (expressed in Quantization Table Sets for an FFV1 encoding) and a range of values that can be divided into sub-ranges proportional to the probability of each symbol encoded. The range is sub-divided until a sub-range can be used to decode the symbol.
```

Comment:
A Range coder encodes a series of binary symbols by using a probability estimation within each context. The sizes of each of the 2 sub ranges are proportional to their estimated probability. The quantization table is used to choose the context used from the surrounding image sample values for the case of coding the sample differences. Coding integers is done by coding multiple binary values. The range decoder will read bytes until it can determine which subrange the input falls into to return the next binary symbol.

Reply:
I updated the PR with this language, taken as is except for using `sub-range` over subrange or sub range. https://github.com/FFmpeg/FFV1/pull/259/commits/a2850a3dd06237144b06eea953b403d5ba07a409